src/componentst/Footer.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { Todo } from '../types/Todo';
+import { Filter } from '../types/Filter';
+import cn from 'classnames';
+
+type Props = {
+  errorMessage: string;
+  todos: Todo[];
+  setFilterBy: Dispatch<SetStateAction<Filter>>;
+  filterBy: Filter;
+  todoCount: number;
+  completedTodos: Todo[];
+  onClearCompleted: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filterBy,
+  setFilterBy,
+  todoCount,
+  completedTodos,
+  onClearCompleted,
+}) => {
+  return (
+    <>
+      {todos.length > 0 && (
```

Comment:
Move this login into the parent component
```suggestion
      {todos.length > 0 && (
```